docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -214,6 +214,9 @@ function processMessage(arg) {
 }
 ```
 
+> [!NOTE]
+> The `showNotification` implementation is not shown in the sample code provided by this article. For an example of how you might implement this function within your add-in, see [Office Add-in Dialog API Example](https://github.com/OfficeDev/Office-Add-in-Dialog-API-Simple-Example).
+
```

Comment:
Two considerations:
- Based on our [internal guidance](https://microsoft.sharepoint.com/teams/OfficePlatform/_layouts/OneNote.aspx?id=%2Fteams%2FOfficePlatform%2FShared%20Documents%2FDevEd%20Content%2FProcedures&wd=target%28Content.one%7C1658CAFF-D22B-44BF-ABF6-4CE62E79C428%2FTips%20%26%20Tricks%2C%20Style%20Guidance%7CB9C53546-5043-4C31-8971-59AF636DCEAE%2F%29
onenote:https://microsoft.sharepoint.com/teams/OfficePlatform/Shared%20Documents/DevEd%20Content/Procedures/Content.one#Tips%20%20Tricks,%20Style%20Guidance&section-id={1658CAFF-D22B-44BF-ABF6-4CE62E79C428}&page-id={B9C53546-5043-4C31-8971-59AF636DCEAE}&end), should we put the note before the code sample (or embellish the lead-in comment)?
- Should we add a comment to the code sample itself? Something before `showNotification` like "// call your add-in's error display logic".

Reply:
Makes sense, though I think adding and in-line code comment wouldn't deviate from the rest of the article.